### ISSUE TITLE
Request for new Keras Metric : Mean Absolute Percentage Error

### ISSUE THREAD

**Issue by remisharoon** [intel-analytics/analytics-zoo#2158]
Now metric MAE is the available regression metric under the package : com.intel.analytics.zoo.pipeline.api.keras.metrics

It would be nice to have more metrics such as "Mean Absolute Percentage Error" and "Mean Squared Error"

**Comment by jason-dai** [comment]
Thanks for the feedback; we'll plan it accordingly.

**Comment by helenlly** [comment]
@hkvision  pls check the feasibility and plan.
